docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
+      
       ```
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/shopping_size/?message=I%20want%20to%20buy%20Large%20shirt%20and%20jeans%20of%2036%20waist&entity_name=shopping_clothes_size&structured_value=&fallback_value=&bot_message='
-      ```
-
-    - *CURL Output:*
+      > **Output **:
 
       ```json
-      {
-        "data": [
-          {
-            "detection": "message",
-            "original_text": "large",
-            "entity_value": {
-              "value": "L"
-            }
-          },
+      {"data": [
           {
-            "detection": "message",
-            "original_text": "36",
-            "entity_value": {
-              "value": "36"
-            }
-          }
-        ]
+              "detection": "fallback_value",
+              "original_text": "9049961794",
+              "entity_value": {"value": "9049961794"},
+              "language": "en"
+          }]
       }
+      
       ```
 
-### passenger_count
+  
 
-- This functionality calls the PassengerDetector class to detect passenger count.
+  ### 5. Email
 
-- Example:
+  > entity_type: email
 
-  - Example 1:
+  The Email Detector has the capability to detect emails within the given text. 
 
-    - ```python
+  **API Example:**
 
-      message = 'Can you please help me to book tickets for 3 people'
-      entity_name = 'no_of_adults'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      ```
+  - **Example 1:  *Detecting emails from message***
 
-    - *Python:*
+    - *Django Shell:*
 
       ```python
-      from ner_v1.chatbot.entity_detection import get_passenger_count
-      output = get_passenger_count(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
+      >>> message = 'my email id is amans.rlx@gmail.com'
+      >>> entity_name = 'email'
+      >>> structured_value = None
+      >>> fallback_value = None
+      >>> bot_message = None
+      
+      >>> from ner_v1.chatbot.entity_detection import get_email
+      >>> output = get_email(message=message,entity_name=entity_name,
+                             structured_value=structured_value,
+                             fallback_value=fallback_value, bot_message=bot_message)
+      >>> print(output)
+      
       ```
 
     - *CURL command:*
 
       ```shell
       URL='localhost'
       PORT=8081
+      
+      curl -i 'http://'$URL':'$PORT'/v1/email/?message=my%20email%20id%20is%20amans.rlx%40gmail.com&entity_name=email&structured_value=&fallback_value=&bot_message='
+      
       ```
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/passenger_count/?message=Can+you+please+help+me+to+book+tickets+for+3+people&entity_name=no_of_adults&structured_value=&fallback_value=&bot_message='
-      ```
-
-    - *CURL Output:*
+      > **Output **
```

Comment:
Formatting inconsistency, Some places it is `**Output**`, some places it is `Output`